src/modality-specific-files/task-events.md

```diff
@@ -263,6 +263,19 @@ and a guide for using macros can be found at
 -->
 {{ MACROS___make_subobject_table("metadata.StimulusPresentation") }}
 
+!!! warning "Important"
+
+    Some metadata of the `StimulusPresentation` object become REQUIRED with the presence of
+    [eye-tracking data](physiological-recordings.md#eye-tracking)
```

Comment:
As mentioned yesterday in the meeting: only for on-screen eyetracking.

Makes the validation of those a bit more complicated (one extra If).

I wonder if we should not have a similar warning in the eyetracking section, or at least a link to the task page.